src/sidebars/favourites/tearout-directive.js

```diff
@@ -162,10 +173,12 @@
                             var _window = currentWindowService.getCurrentWindow();
 
                             var newCardOffset = configService.getTopCardOffset(store.isCompact());
-                            moveWindow(
-                                _window,
-                                currentMousePosition.x - newCardOffset[0] - 2,
-                                currentMousePosition.y - newCardOffset[1] - 1);
+                            _window.setBounds(
```

Comment:
Could this still be a call to moveWindow? 
It looks like the window doesn't change size and this would keep the moveTo / setBounds switch isolated to one function.